### DISCUSSION TITLE
Hey. Guys? Do it best or just delete this repo. So many bugs!

### DISCUSSION THREAD

**Discussion by garawaa** [AvaloniaUI/Avalonia#9839] [General]
Hey. Guys? Do it best or just delete this repo. So many bugs!

**Comment by timunie** [comment]
Not all open issues are bugs, some are questions, others are feature requests. The number of issues is not a good indicator for how good or bad Avalonia is. That said, if you want to know it Avalonia works for you, just try it out. If you find a bug that you think it should be fixed, your help by providing a pull request is very welcome.

**Reply by garawaa** [reply]
Of course I tried it. I think at least it seams not compotable for mobile or web development. So many mobile features are not supperted or does not work.

**Reply by timunie** [reply]
Mobile support was improved very much in latest preview releases. If you want to try mobile support, you really should use nightly builds. 

But yes, our strength is desktop atm. And some features will never work on mobile, for example opening a new window as there is no windowing support on mobile platforms.

**Reply by timunie** [reply]
Filepicker should work in the preview via `TopLevel`. Sensors etc are not an UI feature. You can access them via https://learn.microsoft.com/en-us/xamarin/essentials/ for example. Should work together with Avalonia, but I have not tested it on my own.

**Comment by wj8400684** [comment]
Why don't you write a ui framework yourself?
